jupyterlab-dataops/Dockerfile

```diff
@@ -27,8 +29,9 @@ RUN mkdir -p $HOME && chown -R $USER_NAME $HOME && \
     mkdir -p /mnt/data && chown -R $NB_USER /mnt/data
 
 # Install Python and Jupyter extensions
-RUN pip install -r $HOME/requirements.txt && \
-    jupyter labextension install @jupyterlab/server-proxy jupyterlab-topbar-extension
+RUN pip install -r $HOME/requirements.txt
+RUN if [[ -z "$PROVIDER" ]] ; then echo "Argument PROVIDER not provided" ; else pip install -r $HOME/requirements-${PROVIDER}.txt ; fi
+RUN jupyter labextension install @jupyterlab/server-proxy jupyterlab-topbar-extension
```

Comment:
what is this extension for?

Reply:
To be honest, I don't know. It was here before already :)